tests/align/CMakeLists.txt

```diff
@@ -54,6 +54,33 @@ add_executable (test_align
 # Add dependencies found by find_package (SeqAn).
 target_link_libraries (test_align ${SEQAN_LIBRARIES})
 
+# Add executable for simd tests.
+add_executable(test_align_simd
+               test_align_simd.cpp
+               test_align_simd.h)
+
+# Add dependencies found by find_package (SeqAn).
+target_link_libraries (test_align_simd ${SEQAN_LIBRARIES})
+
+#TODO(rrahn) Enable for windows platforms.
+if (MSVC)
+  # Disable vectorization support on windows for now.
```

Comment:
these should definitely not be deactivated for icc and clang on windows (sync this with #1696 ).

for msvc maybe we can add support in a second PR?